src/sentry/static/sentry/app/views/organizationIntegrations/installedPlugin.tsx

```diff
@@ -16,14 +16,16 @@ import {
 } from 'app/actionCreators/indicator';
 import {PluginNoProject, PluginProjectItem, Organization, AvatarProject} from 'app/types';
 import {SingleIntegrationEvent} from 'app/utils/integrationUtil';
+import space from 'app/styles/space';
+import Switch from 'app/components/switch';
 
 export type Props = {
   api: Client;
   plugin: PluginNoProject;
   projectItem: PluginProjectItem;
   organization: Organization;
   onResetConfiguration: (projectId: string) => void;
-  onEnablePlugin: (projectId: string) => void;
+  onEnablePlugin: (projectId: string, status: boolean) => void;
```

Comment:
@NisanthanNanthakumar If this is going to be used for disabling plugins as well, it needs a new name.